samples/sh/hello-world/get.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+[ -f "$path" ] || _exists=false && match=$(grep "$contentMatch" "$path")
+printf "{\"contentMatch\":\"$match\",\"path\":\"$path\",\"_exists\":$_exists}"
```

Comment:
Missing trailing newline.

```suggestion
printf "{\"contentMatch\":\"$match\",\"path\":\"$path\",\"_exists\":$_exists}"

```